### ISSUE TITLE
Question about Danger

### ISSUE THREAD

**Issue by Eden333** [Farcrada/DangerousDiscordPlugins#2]
Hi, I saw you moved the plugin, and I do understand the selfbotting part, but I'm a bit unclear to what limits.
Personally, I have a small server with friends and from time to time, I move all of us (2 - 5 people at once, myself included).
My question is, would it get me banned if I repeatedly would move us in a faster manner, or would even just the small example I provided be enough to get me banned?

**Comment by Farcrada** [comment]
The bottom line is that the API is used in a faster manner than a user can themselves; this defines "selfbotting" As mentioned: 5 people should not pose an issue; acknowledged one doesn't constantly move all over the server for minutes on end.

Were you to, say move around 10 people here, 10 people there, and do this without giving the API any rest - a delay if you will (which it now does have) - then you are going to be _rate limited_. That may get you marked as abusive and ultimately resort in a Discord ban.

The limits are the API. Of which I'm unsure as well, and seen as I wouldn't wish it upon any user to get banned for additional functionality, I moved the plugin. I'm currently fleshing out delay and warnings since I will admit I got overzealous on that.
